dist/client/app/allDishes/allDishes.html

```diff
@@ -1 +1,87 @@
-<ul class="pagination"><li id="pag__title--current"><h5>All Dishes</h5></li><li class="waves-effect pag__button--next"><a class="pag__button--text" ui-sref="event.guestBill">Your Bill<i class="material-icons">chevron_right</i></a></li></ul><div class="row" id="allDishes" ng-cloak><div class="col s12"><header class="content--centered"><p>Hi, {{::_.capitalize(allDishes.username)}}. <strong class="code">Bill Code: {{ ::allDishes.code }}</strong></p><p>Share your billcode or direct link (<strong>piecemeal.us/{{ ::allDishes.code }}</strong>) with other participants.</p></header><ul class="collection" ng-repeat="dish in allDishes.data.dishes"><li id="dish__item--padding" class="collection-item" ng-if="dish.users.length !== 0">{{ ::_.map(dish.name.split(" "), _.capitalize).join(" ") }} <span class="dish__item--cost">- {{::dish.cost | currency }}</span><br><div class="dish__item--shared">{{ allDishes.getUsersByDish(dish, allDishes.data.users) }} got this.</div><div class="button--share"><div class="button__background waves-effect waves-light btn"><i class="material-icons button--edit share" ng-click="allDishes.shareDish(dish.dish_id, allDishes.user_id, dish.users)">add_circle_outline</i> <i class="material-icons button--edit unshare" ng-if="allDishes.isOnDish(dish.users, allDishes.user_id)" ng-click="allDishes.unshareDish(dish.dish_id, allDishes.user_id, dish.users)">clear</i></div></div></li></ul><a class="button--addDish btn-floating btn-large waves-effect waves-light btn red modal-trigger" data-target="addDish"><i class="material-icons">add</i></a><div id="addDish" class="modal"><div class="modal__box"><h5 class="modal__title">Add A Dish</h5><form role="form" name="dishForm"><div class="modal__input--dish input-field col s8"><input name="name" type="text" placeholder="Dish Name" ng-model="allDishes.dishName" class="dishName" required></div><div class="modal__input--cost input-field col s4"><input name="amount" type="number" min=".01" step="0.01" placeholder="$ 0.00" ng-model="allDishes.amount" class="dishCost" id="addDish" required></div></form><div class="modal--costInfo">Your total so far is <strong><span id="currency-default">{{allDishes.calcUserCurrentTotal(allDishes.data) | currency}}</span></strong>.</div></div><div id="modal-footer" class="modal-footer modal-close"><button type="submit" ng-disabled="dishForm.amount.$error.required || dishForm.name.$error.required || !dishForm.amount.$valid" ng-click="allDishes.addDish(allDishes.dishName, allDishes.amount)" class="addDish btn btn-default">Add Dish</button> <a ui-sref="event.allDishes" class="modal-action modal-close waves-effect waves-green btn-flat">Cancel</a></div></div></div></div>
\ No newline at end of file
+<!-- PAGINATION -->
+<ul class="pagination">
```

Comment:
The htmlmin grunt task was doing something weird and the app wouldn't load with it on so I took it out...  Not sure why.  You could try setting it up to work again.